recipes/quazip/all/conanfile.py

```diff
@@ -48,10 +48,10 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("qt/5.15.9")
-        self.requires("zlib/1.2.13")
+        self.requires("qt/[>=5.15.2]")
```

Comment:
Does this mean that this package is Qt 6 compatible?

Reply:
Thank you for the review! Yes it is compatible. I'm just wondering which qt version will be picked up in the test.

That fact that the target name contains the qt version can be problematic though.